runtime/bindings/python/tests/test_ngraph/test_node_factory.py

```diff
@@ -5,7 +5,6 @@
 import openvino.opset8 as ov
 from openvino.exceptions import UserInputError
 from openvino.utils.node_factory import NodeFactory
```

Comment:
It is later used as `_NodeFactory`, why?

Reply:
_pyngraph incorrect name in tests, need another name of library

Reply:
in openvino I did not find the name Nodefactory, only in pyopenvino directly